lib/net/server/rlpxserver.js

```diff
@@ -116,6 +118,20 @@ class RlpxServer extends Server {
     await super.stop()
   }
 
+  /**
+   * Return Rlpx info
+   */
+  getRlpxInfo () {
+    const id = this.rlpx._id.toString('hex')
+    return {
+      enode: `enode://${id}@[${this.ip}]:${this.port}`,
+      id: id,
+      ip: this.ip,
+      listenAddr: `[${this.ip}]:${this.port}`,
+      ports: { 'discovery': this.port, 'listener': this.port }
+    }
```

Comment:
Values already added all look good.

Reply:
Update: ah, just seeing that the missing values (`name`, `protocols`) are collected elsewhere.

Reply:
Another look: `ports` are currently returned as strings, can we have these as numbers? Just had a short look, am a bit confused how this information is lost, `getNodeInfo()` returns these as numbers.

Reply:
(ah, so to be precise: this is rather a comment for the RPC method)

Reply:
I'll check it